src/main/java/bio/terra/service/StudyService.java

```diff
@@ -35,6 +37,11 @@ public StudyService(StudyDao studyDao, Stairway stairway, DatasetService dataset
         this.datasetService = datasetService;
     }
 
+    /**
+     * Create a study
+     * @param studyRequest from the REST API
+     * @return a StudySummaryModel
+     */
```

Comment:
I'm of the opinion that we should use names that are self documenting so we don't need boiler plate documentation. The swagger is the documentation for the cli. We should only add documentation when the code is doing something surprising so when documenting additional functionality that needs to be added. I guess we need a team discussion.

Reply:
Yes, I agree. I added this in when I had more to say and didn't remove it when I had nothing to say. I'll remove.